src/DependencyInjection/SonataUserExtension.php

```diff
@@ -195,39 +181,29 @@ public function configureClass($config, ContainerBuilder $container): void
         $container->setParameter(sprintf('sonata.user.admin.group.%s', $modelType), $config['class']['group']);
     }
 
-    /**
-     * @param array            $config
-     * @param ContainerBuilder $container
-     */
     public function configureAdminClass($config, ContainerBuilder $container): void
     {
         $container->setParameter('sonata.user.admin.user.class', $config['admin']['user']['class']);
         $container->setParameter('sonata.user.admin.group.class', $config['admin']['group']['class']);
     }
 
-    /**
-     * @param array            $config
-     * @param ContainerBuilder $container
-     */
     public function configureTranslationDomain($config, ContainerBuilder $container): void
     {
         $container->setParameter('sonata.user.admin.user.translation_domain', $config['admin']['user']['translation']);
         $container->setParameter('sonata.user.admin.group.translation_domain', $config['admin']['group']['translation']);
     }
 
-    /**
-     * @param array            $config
```

Comment:
you removed this but didn't add the typehint in the signature

Reply:
Fixed!